packages/at_chops/lib/src/at_chops_base.dart

```diff
@@ -0,0 +1,54 @@
+import 'dart:typed_data';
+
+import 'package:at_chops/src/algorithm/at_algorithm.dart';
+import 'package:at_chops/src/algorithm/at_iv.dart';
+import 'package:at_chops/src/algorithm/default_encryption_algo.dart';
+import 'package:at_chops/src/algorithm/default_signing_algo.dart';
+import 'package:at_chops/src/algorithm/default_hashing_algo.dart';
+import 'package:at_chops/src/key/signing_key.dart';
+
+/// Base class for all Cryptographic and Hashing Operations. Callers have to either implement
```

Comment:
@VJag please review dart docs for this file which is the entry point for this package